index.js

```diff
@@ -2,18 +2,14 @@ require('dotenv').config()
 
 const Discord = require('discord.js')
 const nemojis = require('node-emoji')
-const get = require('get-value')
 const client = new Discord.Client()
 
 client.on('message', msg => {
-  const members = get(msg, 'mentions.members')
-  if (!members) return
-
-  const jason = members.find(member => member.user.username === 'jasonetco')
-  if (!jason) return
+  if (!msg.mentions.users.has(client.user)) return
```

Comment:
If `msg.mentions` or `msg.mentions.users` is undefined, this will throw - unless the Discord SDK fills them in as empty Maps? Might be worth adding checks for those:

```suggestion
  if (!(msg.mentions && msg.mentions.users && msg.mentions.users.has(client.user))) return
```

That's why `get-value` was used in the first place.

Reply:
Discord.js does return empty collections (Note the lack of a ? to mean nullable) https://discord.js.org/#/docs/main/stable/class/MessageMentions?scrollTo=users

Reply:
Ah got it - thanks for the clarification!